### PR TITLE
change dropdown trigger & menu width

### PR DESCRIPTION
**Pull request by ubergesundheit**
Fixes giantswarm/giantswarm#13502

This PR:

- restricts the maximum length of the organization dropdown trigger
- allows maximum length of the organization dropdown menu

<details>

<summary>Screenshots</summary>

image

image

</details>

### CODE REVIEW

File: src/components/UI/Navigation/OrganizationDropdown.js [hunk 1]
```diff
@@ -145,6 +147,18 @@ const OrganizationDropdown = ({
...
+
+  if (sortedOrganizations.length !== 0) {
+    if (selectedOrganization.length > TriggerLabelMaxLength) {
+      triggerLabel = (
+        <>{selectedOrganization.slice(0, TriggerLabelMaxLength)}&hellip;</>
+      );
+    } else {
+      triggerLabel = selectedOrganization;
+    }
+  }
```
Comment (ubergesundheit) [inline src/components/UI/Navigation/OrganizationDropdown.js:160]: Tried to do this with css overflow and ellipsis but couldn't make it working. The component uses many inline divs. This was quick and easy

Comment (axbarsan) [reply src/components/UI/Navigation/OrganizationDropdown.js:160]: We actually have an element made specifically for this
https://github.com/giantswarm/happa/blob/a939dc81043fab10d0ee676f9cfc4927fd25c355/src/components/UI/Truncated.tsx

Comment (ubergesundheit) [reply src/components/UI/Navigation/OrganizationDropdown.js:160]: :facepalm: I was searchting for this.. I tried to look for overflow or ellipsis :sweat_smile:

**Review by axbarsan — APPROVED** [review]
LGTM

**Review by marians — APPROVED** [review]
Looking good!

We have a limit on org name length, right? So what we see there is pretty much the worst case.

**Comment by ubergesundheit** [comment]
> We have a limit on org name length, right? So what we see there is pretty much the worst case.

Yes 63 characters is the upper limit

**Pull request merged**